test/byContract/IexecAccessors/IexecAccessors.ts

```diff
@@ -0,0 +1,117 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { deployments, ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import { IexecInterfaceNative, IexecInterfaceNative__factory } from '../../../typechain';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+/**
+ * Test constant view functions.
+ */
```

Comment:
Can you remove admin & iexecPoco please?

Reply:
[Keep only pocoAsAnyone instance](https://github.com/iExecBlockchainComputing/PoCo/pull/96/commits/7d814b0c25a874d8e66430b55c909c7bd1febb6b)